src/Lykke.AlgoStore.Api/Models/DeleteAlgoModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Lykke.AlgoStore.Api.Models
+{
+    public class DeleteAlgoModel
+    {
+        [Required]
+        public string AlgoId { get; set; }
+        [Required]
+        public string AlgoClientId { get; set; }
```

Comment:
do we need algoClientId? Check with Mira if an administrator can delete algos of other users.

Reply:
Right now every user can delete his/her own algos.